src/vm/ceeload.cpp

```diff
@@ -2139,7 +2139,7 @@ void Module::BuildStaticsOffsets(AllocMemTracker *pamTracker)
                         (S_SIZE_T(2 * sizeof(DWORD))*(S_SIZE_T(dwNumTypes)+S_SIZE_T(1)))));
 
                 for (DWORD i = 0; i < dwIndex; i++) {
-                    pRegularStaticOffsets[i * 2    ] = dwGCHandles[0]*sizeof(OBJECTREF);
```

Comment:
`sizeof(OBJECTREF)` can be just `TARGET_POINTER_SIZE`. We use `TARGET_POINTER_SIZE` for object reference sizes in many places already, so a few more are not a concern.

Reply:
Sure, I will replace with `TARGET_POINTER_SIZE` instead